unittests/InstructionCountCI/AFP/VEX_map1.json

```diff
@@ -30,13 +30,14 @@
       ]
     },
     "vrsqrtss xmm0, xmm1, xmm2": {
-      "ExpectedInstructionCount": 4,
+      "ExpectedInstructionCount": 5,
       "Comment": [
         "FEAT_FPRES could make this more optimal",
         "Map 1 0b10 0x52 128-bit"
       ],
       "ExpectedArm64ASM": [
         "mov v16.16b, v17.16b",
+        "mov v0.16b, v16.16b",
```

Comment:
This mov looks extraneous. The `fmov s0, #0x70 (1.0000)` afterwords will always zero the full register even with AFP.NEP. Same situations for the other locations that add this move.

Reply:
This specific case looks like:
```
   "vrsqrtss xmm0, xmm1, xmm2": {
      "ExpectedInstructionCount": 5,
      "Comment": [
        "FEAT_FPRES could make this more optimal",
        "Map 1 0b10 0x52 128-bit"
      ],
      "ExpectedArm64ASM": [
        "mov v16.16b, v17.16b",
        "mov x0, #0x3f800000",
        "mov v16.s[0], w0",
        "fsqrt s1, s18",
        "fdiv s16, s16, s1"
      ]
    },
```

By inserting 0x3f80000 (1.0f) directly into the bottom of v16, we can then use that as the destination and first source of fdiv. WDYT?

Reply:
This looks ok:

```
        "mov v16.16b, v17.16b",
        "fmov s0, #0x70 (1.0000)",
        "fsqrt s1, s18",
        "fdiv s0, s0, s1",
        "mov v16.s[0], v0.s[0]"
```

Reply:
Yea, that matches expectations for my mental model.